tests/unit/lms/services/digest_test.py

```diff
@@ -139,10 +150,21 @@ def sender(self):
 
     @pytest.fixture
     def h_api(self, h_api):
+        def asdict(annotation):
+            return {
+                "author": {
+                    "userid": annotation.userid,
+                },
+                "group": {
+                    "authority_provided_id": annotation.authority_provided_id,
+                },
+            }
```

Comment:
Here I want to use `AnnotationFactory()` because it's convenient but I actually have to convert the resulting `Annotation` models back into the dict format that the h API returns. Could add a comment here...

Reply:
Done